src/Image.svelte

```diff
@@ -1,10 +1,23 @@
 <script>
-  /**
-   * Cloudinary Image Component
-   */
+    /**
+     * Cloudinary Image Component
+     */
+    import {onMount, afterUpdate} from 'svelte';
+    import {getImageTag, responsive} from "./utils";
+    const bindImage = (node) => (imgElement = node);
 
-  import { getImageTag } from "./utils";
-  $: imageTag = getImageTag($$props);
+    let imgElement; // Reference to underlying <img> element
+    let isResponsive;
+
+    $: attributes = getImageTag($$props).attributes();
```

Comment:
Something is not very clear to me... you get the Image tag, take its attributes, but at the end the attributes of the img below is based on the same `attributes`. I probably miss something

Reply:
Added comments to the code